frontend/src/components/organisms/MatchesList/MatchesList.tsx

```diff
@@ -20,6 +24,9 @@ export const MatchesList = ({tournamentId}: MatchesListProps) => {
 
    const [expanded, setExpanded] = React.useState<string | boolean>(false);
    const [matchesListItems, setMatchesListItems] = React.useState<MatchListItem[] | undefined>();
+   const [matchesInformationListDto, setMatchesInformationListDto] = React.useState<MatchInformationDto[] | undefined>(undefined);
```

Comment:
Nazwa Dto raczej nam nie potrzebna. Na forncie mamy praktycznie same Dto :) 
I raz zapisujesz tak a raz tak.

Reply:
ok, przemyślę, bo trochę mi to pomaga odnaleźć. 

![image](https://user-images.githubusercontent.com/41823689/114519613-df1d2000-9c40-11eb-8f25-e4dec01dddbd.png)
To też zmieniłbym na Dto